src/ideations/ideations.controller.ts

```diff
@@ -0,0 +1,106 @@
+import {
+    Controller,
+    Get,
+    Param,
+    Post,
+    Body,
+    Patch,
+    Delete,
+    ParseIntPipe
+} from "@nestjs/common";
+import { IdeationsService } from "./ideations.service";
+import { CreateIdeationDto } from "./dto/create-ideation.dto";
+import { UpdateIdeationDto } from "./dto/update-ideation.dto";
+import { ApiCreatedResponse, ApiTags } from "@nestjs/swagger";
+import { Ideation } from "./entities/ideation.entity";
+import { CreateIdeationVoteDto } from "./dto/create-ideation-vote.dto";
+
+@Controller()
+@ApiTags("ideations")
+export class IdeationsController {
+    constructor(private readonly ideationsService: IdeationsService) {}
+
+    //TODO updated userId to grab uuid/JWT once 
+    @Post("users/:userId/teams/:teamId/ideations")
+    @ApiCreatedResponse({type: Ideation})
+    createIdeation(
+        @Param("userId") userId: string,
+        @Param("teamId", ParseIntPipe) teamId: number,
+        @Body() createIdeationDto: CreateIdeationDto,
+    ) {
+        return this.ideationsService.createIdeation(
+            teamId, 
+            userId, 
+            createIdeationDto
+        );
+    };
+
+    @Get("/teams/:teamId/ideations")
+    @ApiCreatedResponse({type: Ideation})
+    async getIdeationsByVoyageTeam(
+        @Param("teamId", ParseIntPipe) teamId: number
+    ) {
+        const projectIdeas = await this.ideationsService.getIdeationsByVoyageTeam(teamId);
+        return projectIdeas.map((projectIdea)=>{
```

Comment:
I don't think this is necessary. We need the user associated with the vote so that data will be returned to the fe anyways and the length can easily be calculated. Also, in general, you should not be adding any logic in the controller.

Reply:
Sorry I didn't really highlight the proper part. I'm referring to the map and returning the project vote count length.

Reply:
based on the the confluence requirements they wanted the count of votes along with member info. Just want to confirm you don't want vote count before editing it out.

Reply:
Let's see what others think first.

Reply:
There you have it. So you can probably just revert that back to what it was before.